cllc-public-app/ViewModels/Application.cs

```diff
@@ -545,6 +546,6 @@ public class Application
         [JsonConverter(typeof(StringEnumConverter))]
         public Zoning? LgZoning { get; set; }
 
-
+        public List<CapacityArea> ServiceAreas { get; set; }
```

Comment:
I see this in the Application Summary client side model, but was it also added to the application client side model?  If not I think it should be added there (and that may avoid the need to do extra casting)

Reply:
hm interesting you're right i did miss it in on the application model. interesting that it seems to work without it. will add it there and see if that helps with my issue